engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -604,6 +604,8 @@ protected void serviceInit(Configuration conf) throws Exception
   }
 
   public static final String PLUGINS_CONF_KEY = "apex.plugin.stram.plugins";
```

Comment:
@tushargosavi @PramodSSImmaneni @chinmaykolhatkar Please rename to "apex.engine.plugins". My concern is using plugin twice and referring to stram.